src/tools/react-native.ts

```diff
@@ -175,22 +171,3 @@ export async function renameReactNativeApp(
     }),
   )
 }
-
-async function moveDeepFolder(toolbox: GluegunToolbox, oldPath: string, newPath: string) {
-  const { filesystem } = toolbox
-  const { path } = filesystem
-
-  // ensure newPath exists
-  await filesystem.dirAsync(newPath)
-
-  // move the files
-  const filesToMove = children(oldPath)
-  const movePromises = filesToMove.map((file) =>
-    filesystem.moveAsync(path(oldPath, file), path(newPath, file)),
-  )
```

Comment:
When you had a bundle identifier like `com.infinitered.pizzaapp`, the files we're copied over correctly (maybe because we're using a much older version of `GlueGun`, and consequently `fs-jetpack`).